test_inventories.py

```diff
@@ -1,13 +1,23 @@
 import unittest
+from unittest.mock import MagicMock
+from inventory import Fetcher
 from inventory import InventoryBuilder
+import test_mocks as mocks
```

Comment:
not sure this is the best naming for it. it's confusing that you are importing `fixtures` and then have a `mock` type that is just returning fixtures. try to find a better name for this or a better structure

`mocks` are test doubles and not fixtures they're actually Classes as is MagicMock, having this name here will make it confusing for the reader